arcgis-ios-sdk-samples/Features/Feature layer rendering mode (map)/FeatureLayerRenderingModeMapViewController.swift

```diff
@@ -20,7 +20,7 @@ class FeatureLayerRenderingModeMapViewController: UIViewController {
     @IBOutlet weak var dynamicMapView: AGSMapView!
     @IBOutlet weak var staticMapView: AGSMapView!
     
-    var _zoomed = false
+    private var isZoomed = false
```

Comment:
I think this should just be `zoomed` since it is mutable. From the Swift API Design Guidelines:

> Uses of Boolean methods and properties should read as assertions about the receiver when the use is nonmutating